presto-main/src/main/java/com/facebook/presto/sql/planner/optimizations/TransformQuantifiedComparisonApplyToScalarApply.java

```diff
@@ -160,27 +174,43 @@ private PlanNode rewriteQuantifiedEqualsAllApplyNode(ApplyNode node, QuantifiedC
 
             Symbol quantifiedComparisonSymbol = getOnlyElement(node.getSubqueryAssignments().getSymbols());
             LogicalBinaryExpression valueComparedToSubquery = new LogicalBinaryExpression(LogicalBinaryExpression.Type.AND,
-                    new ComparisonExpression(EQUAL, minValue.toSymbolReference(), maxValue.toSymbolReference()),
+                            new ComparisonExpression(EQUAL, minValue.toSymbolReference(), maxValue.toSymbolReference()),
                     new ComparisonExpression(EQUAL, quantifiedComparison.getValue(), minValue.toSymbolReference())
             );
             return projectExpressions(applyNode, Assignments.of(quantifiedComparisonSymbol, valueComparedToSubquery));
         }
 
         private PlanNode rewriteQuantifiedOrderableApplyNode(ApplyNode node, QuantifiedComparisonExpression quantifiedComparison, RewriteContext<PlanNode> context)
         {
-            QualifiedName aggregationFunction = chooseAggregationFunction(quantifiedComparison);
             PlanNode subqueryPlan = context.rewrite(node.getSubquery());
+
+            QualifiedName count = QualifiedName.of("count");
```

Comment:
Consider making this private static final of the class to express that you relay on syntax at this point.